src/application/management.rs

```diff
@@ -31,3 +32,10 @@ fn build_health_router<H: Health>(health_indicator: Option<H>) -> Option<Router>
             .layer(Extension(health_indicator)),
     )
 }
+
+fn build_metrics_router() -> Option<Router> {
+    Some(Router::new().route(
+        "/metrics",
+        get(move || std::future::ready(crate::get_metrics())),
```

Comment:
We can return here just Router, it is not Option<Router>
I guess there is no need for move, also you can use here 
`|| async {get_metrics()}`
but you still can keep future::ready()